mlflow/langchain/output_parsers.py

```diff
@@ -99,3 +101,36 @@ def _type(self) -> str:
 
     def parse(self, text: str) -> dict[str, Any]:
         return asdict(StringResponse(content=text))
+
+
+@experimental
+class ChatAgentOutputParser(BaseTransformOutputParser[str]):
+    """
+    OutputParser that wraps the string output into a dictionary representation of a
+    :py:class:`ChatAgentResponse`
+    """
+
+    @classmethod
+    def is_lc_serializable(cls) -> bool:
+        """Return whether this class is serializable."""
+        return True
+
+    @property
+    def _type(self) -> str:
+        """Return the output parser type for serialization."""
+        return "mlflow_chat_agent"
+
+    def parse(self, text: str) -> dict[str, Any]:
+        """Returns the input text as a ChatAgentResponse."""
```

Comment:
```suggestion
        """Returns the output text as a dictionary representation of a :py:class:`ChatAgentResponse`."""
```

Could you also document the reason why we return dictionary not `ChatAgentResponse`?